### DISCUSSION TITLE
CLIPVisionModelWithProjection crash

### DISCUSSION THREAD

**Discussion by aareschluchtje** [azuritecoin/OnnxDiffusersUI#38] [Q&A] [answered]
Hi, after updating to the newest version I got some errors when trying to generate an image.
Before the update (yesterday) it seemed to work fine. Also fully uninstalling it and installing the latest version did not work.
![image](https://user-images.githubusercontent.com/9335763/203651678-7e12f581-49fb-469d-826f-09562bc172d2.png)

Do you have any idea why this could be the case?

**Comment by azuritecoin** [comment, marked as answer]
`diffusers` has updated very recently, v0.8.0 was released hours ago. You will need to update transformers to the dev version:  
`pip install git+https://github.com/huggingface/transformers`
